packages/integrations/stores/gcp-secret-manager-config-store/src/gcp-secret-manager.ts

```diff
@@ -0,0 +1,65 @@
+import { SecretManagerServiceClient } from '@google-cloud/secret-manager';
+import { KeyValueConfigStore } from '@configu/ts';
```

Comment:
`@configu/ts` has been replaced by `@configu/sdk`.

Reply:
sorry  for missleading, thenew version of KeyValueConfigStore should come from:
```import { KeyValueConfigStore } from '@configu/integrations/src/utils/KeyValue';```

Reply:
fixed